apps/tangle-dapp/src/containers/bridge/BridgeContainer.tsx

```diff
@@ -836,7 +827,7 @@ const BridgeContainer = () => {
                   placeholder="Enter amount to bridge"
                   wrapperClassName="dark:bg-mono-180"
                   showMaxAction
-                  decimals={selectedToken.decimals}
+                  decimals={selectedToken?.decimals ?? 18}
```

Comment:
selectedToken?.decimals

Remove `?? 18` since there's already a default value for this prop.